### PR TITLE
Migrate to Github Actions

### PR DESCRIPTION
**Pull request by thomasferrandiz**
Issue: https://github.com/rancher/ecm-distro-tools/issues/390

### CODE REVIEW

File: .github/workflows/image-push.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
```
Comment (manuelbuil) [inline .github/workflows/image-push.yml:6]: The permission to read EIO stuff is missing

File: Makefile [hunk 2]
```diff
@@ -16,38 +16,38 @@ PKG ?= github.com/k8snetworkplumbingwg/whereabouts
 SRC ?= github.com/k8snetworkplumbingwg/whereabouts
 TAG ?= v0.6.3$(BUILD_META)
 
-ifneq ($(DRONE_TAG),)
-	TAG := $(DRONE_TAG)
-endif
-
 ifeq (,$(filter %$(BUILD_META),$(TAG)))
-	$(error TAG needs to end with build metadata: $(BUILD_META))
+	$(error TAG $(TAG) needs to end with build metadata: $(BUILD_META))
```
Comment (manuelbuil) [inline Makefile:20]: I think you need to move this to the left

**Review by manuelbuil — APPROVED** [review]

File: .github/workflows/image-push.yml (hunk 1 above)
Comment (thomasferrandiz) [reply .github/workflows/image-push.yml:6]: the permissions are just below inside the `jobs.push-multiarch` section

Comment (manuelbuil) [reply .github/workflows/image-push.yml:6]: sorry, you are right!

**Review by manuelbuil — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (thomasferrandiz) [reply Makefile]: done!

**Pull request merged**